### PR TITLE
Add HoCoRT

### PR DESCRIPTION
**Pull request by ignasrum**
Add HoCoRT. This is a tool for removing specific organisms from sequencing reads.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/hocort/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (bgruening) [inline recipes/hocort/LICENSE:1]: If you have a LICENSE file in your tarball then this file here is not needed.

File: recipes/hocort/meta.yaml [hunk 2]
```diff
@@ -45,7 +45,6 @@ test:
 about:
   home: https://github.com/ignasrum/hocort
   license: MIT
-  license_file: LICENSE
```
Comment (bgruening) [inline recipes/hocort/meta.yaml]: No, please leave this in here. Conda will get the file named LICENSE from your tarball

Comment (ignasrum) [reply recipes/hocort/meta.yaml]: Sorry, I thought it referred to the LICENSE file that I removed.

**Comment by ignasrum** [comment]
I was throwing stuff at the wall and made a bunch of unnecessary commits. After I found #183, I managed to fix everything. Should I make a new clean pull request or keep this one?

**Comment by bgruening** [comment]
We can squash your commits if you are ok with that.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@BiocondaBot please fetch artifacts

**Comment by ignasrum** [comment]
Yeah, you can squash them, that's fine.

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Pull request merged**